app/views/admin/conferences/show.html.haml

```diff
@@ -44,41 +44,40 @@
 
 .row
   .col-md-12
-    .row
-      .col-md-8
-        .row#registrations
-          .col-md-12
-            = render partial: 'line_chart', locals: { title: 'Registrations over time',
-                                                       name: 'registrations',
-                                                       conferences: [@conference],
-                                                       active_conferences: [@conference],
-                                                       deactive_conferences: [],
-                                                       y: @registrations,
-                                                       x: @registration_weeks,
-                                                       unit: 'weeks' }
-        .row#submissions
-          .col-md-12
-            = render partial: 'line_chart', locals: { title: 'Submissions over time',
-                                                        name: 'submissions',
-                                                        conferences: @submissions,
-                                                        active_conferences: @submissions,
-                                                        deactive_conferences: [],
-                                                        y: @submissions_data,
-                                                        x: @cfp_weeks,
-                                                        unit: 'weeks' }
-      .col-md-4
-        = render partial: 'todo_list', locals: { conference_progress: @conference_progress,
-                                                 conference: @conference }
-      .row#tickets
-        .col-md-8
-          = render partial: 'line_chart', locals: { title: 'Tickets sold over time',
-                                                      name: 'tickets',
-                                                      conferences: @tickets,
-                                                      active_conferences: @tickets,
+    .col-md-8
+      .row#registrations
```

Comment:
What's this got to do with the openID buttons?

Reply:
It was causing unwanted horizontal scroll in the page. I thought may be I should just solve this small error in this pr as well :sweat_smile: